.github/workflows/ci.yml

```diff
@@ -4,46 +4,26 @@ on:
   push:
     branches: [main]
   pull_request:
-    types: [opened, reopened, ready_for_review, edited, synchronize]
 
 jobs:
   test-python35:
-    if: github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tox-env:
-          - py35-codetest-noorgs
-          - py35-codetest-withorgs
+        tox-env-suffix:
+          - codetest-noorgs
+          - codetest-withorgs
+        python-version:
+          - 3.5
+          - 3.8
     steps:
      - uses: actions/checkout@v2
-     - name: Set up Python 3.5
+     - name: Set up Python
        uses: actions/setup-python@v2
        with:
-         python-version: 3.5
+         python-version: ${{ matrix.python-version }}
      - name: Install dependencies
        run: |
          pip install tox
      - name: Test with tox
-       run: tox -e ${{ matrix.tox-env }}
-
-  test-python38:
-    if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        tox-env:
-          - py38-codetest-noorgs
-          - py38-codetest-withorgs
-          - py38-codelint
```

Comment:
Thank you @OmarIthawi . I believe this line slipped out from your hands. I'll restore it